Userland/Libraries/LibJS/Runtime/Temporal/Instant.cpp

```diff
@@ -183,14 +184,26 @@ ThrowCompletionOr<BigInt*> add_instant(VM& vm, BigInt const& epoch_nanoseconds,
     return result;
 }
 
-// 8.5.7 DifferenceInstant ( ns1, ns2, roundingIncrement, smallestUnit, roundingMode ), https://tc39.es/proposal-temporal/#sec-temporal-differenceinstant
-BigInt* difference_instant(VM& vm, BigInt const& nanoseconds1, BigInt const& nanoseconds2, u64 rounding_increment, StringView smallest_unit, StringView rounding_mode)
+// 8.5.7 DifferenceInstant ( ns1, ns2, roundingIncrement, smallestUnit, largestUnit, roundingMode ), https://tc39.es/proposal-temporal/#sec-temporal-differenceinstant
+TimeDurationRecord difference_instant(VM& vm, BigInt const& nanoseconds1, BigInt const& nanoseconds2, u64 rounding_increment, StringView smallest_unit, StringView largest_unit, StringView rounding_mode)
 {
+    auto n2_minus_n1 = nanoseconds2.big_integer().minus(nanoseconds1.big_integer());
+
     // 1. Assert: Type(ns1) is BigInt.
     // 2. Assert: Type(ns2) is BigInt.
 
-    // 3. Return ! RoundTemporalInstant(ns2 - ns1, roundingIncrement, smallestUnit, roundingMode).
-    return round_temporal_instant(vm, *js_bigint(vm, nanoseconds2.big_integer().minus(nanoseconds1.big_integer())), rounding_increment, smallest_unit, rounding_mode);
+    // 3. Assert: The following step cannot fail due to overflow in the Number domain because abs(ns2 - ns1) ≤ 2 × nsMaxInstant.
+    VERIFY(n2_minus_n1.absolute_value() <= ns_max_instant.multiplied_by("2"_sbigint));
```

Comment:
So I think this can just become 
`VERIFY(n2_minus_n1.unsigend_value() <= ns_max_instant.multiplied_by("2"_sbigint));`

Reply:
![image](https://user-images.githubusercontent.com/7570033/196235442-50e21b47-0f59-46cb-86ce-2d3339d9eca8.png)

I think the comparisons aren't implemented yet :')

Reply:
Whoops, didn't look correctly. Yes `<=` doesn't exist but `<` and `==` do so yeah you can just implement that one.